### ISSUE TITLE
ruby opt/linux builds

### ISSUE THREAD

**Issue by apolcyn** [grpc/grpc#8124]
these seem to be failing almost all of the time right now on jenkins

**Comment by kpayson64** [comment]
@apolcyn Is this still the case?hh

**Comment by apolcyn** [comment]
the error on jenkins builds appears to be:
```
make: *** No rule to make target `/tmp/libs/opt/grpc-1.dll'.  Stop.
rake aborted!
```
this is on the linux platform. 

locally though, running `tools/run_tests/build_artifact_ruby.sh` on linux (the platform and script leading to that error) seems to work fine.

**Comment by apolcyn** [comment]
to update, this actually seems to only be an issue on the master branch

**Comment by apolcyn** [comment]
this was fixed by #8957
